test/fixtures/url-tests.js

```diff
@@ -5110,6 +5110,246 @@ module.exports =
     "search": "?test",
     "hash": "#x"
   },
+  "# File URLs and many (back)slashes",
```

Comment:
Ditto, although this file is missing https://github.com/w3c/web-platform-tests/blob/3eff1bd564a0085347d31d80fbee13aaa2e3c2ca/url/urltestdata.json#L5380-L5451 ?